Godot_test/ClientScripts/TopMenu.cs

```diff
@@ -1,26 +1,59 @@
 using Godot;
+using Roguelike.Abstract;
 using System;
+using System.IO;
 
 public partial class TopMenu : Control
 {
-	private async void _on_exit_pressed()
+  public override void _Ready()
+  {
+		string path = System.IO.Path.GetTempPath() + "Roguelike";
+	string[] folders = Directory.GetDirectories(path);
+		int numberOfSaves = folders.Length;
+	int folderCount = folders.Length;
+	if (folderCount == 0)
+		{
+	  var button = (Button)GetNode("VBoxContainer/Continue");
+			button.Visible = false;
+	}
+			
+
+	var game = (Game)GetNode("/root/Game");
+		
+  }
+
+  private async void _on_exit_pressed()
 	{
-		var anim = (AnimationPlayer)GetNode("Exit/AnimationPlayer");
+		var anim = (AnimationPlayer)GetNode("VBoxContainer/Exit/AnimationPlayer");
 		anim.Play("clicked");
 		await ToSignal(GetTree().CreateTimer(0.4),"timeout");
 		GetTree().Quit();
 	}
 
   private async void _on_start_pressed()
   {
-		var startButton = (Button)GetNode("Start");
+		var startButton = (Button)GetNode("VBoxContainer/Start");
 		startButton.Disabled= true;
-		var anim = (AnimationPlayer)GetNode("Start/AnimationPlayer");
+		var anim = (AnimationPlayer)GetNode("VBoxContainer/Start/AnimationPlayer");
 		anim.Play("clicked");
 		await ToSignal(GetTree().CreateTimer(0.4), "timeout");
 		var game = (Game)GetNode("/root/Game");
 		game.GenerateDungeon();
 		var parent = (CanvasLayer)GetParent();
 		parent.Visible = false;
   }
+
+  private async void _on_continue_pressed()
```

Comment:
A lot of similar code as in _on_start_pressed, recactror comon code to the method and used in both places.